cg-producer/pypi-readme.md

```diff
@@ -0,0 +1,23 @@
+Library for generating PyPI Call Graphs through PyCG
```

Comment:
Please use `#` to describe section headers. Makes it more readable. (e.g. `#` is Header1, `##` is Header2 etc.)